curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/65802ee9706eb103aea442f8.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 65802ee9706eb103aea442f8
+title: Task 103
+challengeType: 19
+dashedName: task-103
+---
+
+# --description--
+
+The phrase `I'd love to join` is used with the contraction `I'd`, which is short for `I would`, expressing willingness and enthusiasm.
+
+# --question--
+
+## --text--
+
+Choose the correct expanded form of the contraction `I'd` in the given context.
+
+`Really? That sounds great! _.`
```

Comment:
```suggestion
`Really? That sounds great! ...`
```
I'm still not 100% sure this is the best approach but this could be a way to fix this hybrid task.